src/main/java/org/venice/beachfront/bfapi/geoserver/GeoserverEnvironment.java

```diff
@@ -0,0 +1,275 @@
+			"                    </nativeBoundingBox>\n" + 
+			"                    <metadata>\n" + 
+			"                        <entry key=\"JDBC_VIRTUAL_TABLE\">\n" + 
+			"                            <virtualTable>\n" + 
+			"                                <name>" + LAYER_NAME + "</name>\n" + 
+			"                                <sql>\n" + 
+			"                                    SELECT * FROM __beachfront__geoserver\n" + 
+			"                                     WHERE ('%jobid%' = '' AND '%sceneid%' = '')\n" + 
+			"                                        OR (job_id = '%jobid%')\n" + 
+			"                                        OR (scene_id = '%sceneid%')\n" + 
+			"                                </sql>\n" + 
+			"                                <escapeSql>false</escapeSql>\n" + 
+			"                                <keyColumn>job_id</keyColumn>\n" + 
+			"                                <geometry>\n" + 
+			"                                    <name>geometry</name>\n" + 
+			"                                    <type>Geometry</type>\n" + 
+			"                                    <srid>4326</srid>\n" + 
+			"                                </geometry>\n" + 
+			"                                <parameter>\n" + 
+			"                                    <name>jobid</name>\n" + 
+			"                                    <regexpValidator>^(%|[a-f0-9]\\{\\{8\\}\\}-[a-f0-9]\\{\\{4\\}\\}-[a-f0-9]\\{\\{4\\}\\}-[a-f0-9]\\{\\{4\\}\\}-[a-f0-9]\\{\\{12\\}\\})$</regexpValidator>\n" + 
+			"                                </parameter>\n" + 
+			"                                <parameter>\n" + 
+			"                                    <name>sceneid</name>\n" + 
+			"                                    <regexpValidator>^\\w+:\\w+$</regexpValidator>\n" + 
+			"                                </parameter>\n" + 
+			"                            </virtualTable>\n" + 
+			"                        </entry>\n" + 
+			"                        <entry key=\"time\">\n" + 
+			"                            <dimensionInfo>\n" + 
+			"                                <enabled>false</enabled>\n" + 
+			"                                <attribute>time_of_collect</attribute>\n" + 
+			"                                <presentation>CONTINUOUS_INTERVAL</presentation>\n" + 
+			"                                <units>ISO8601</units>\n" + 
+			"                                <defaultValue>\n" + 
+			"                                    <strategy>FIXED</strategy>\n" + 
+			"                                    <referenceValue>P1Y/PRESENT</referenceValue>\n" + 
+			"                                </defaultValue>\n" + 
+			"                            </dimensionInfo>\n" + 
+			"                        </entry>\n" + 
+			"                    </metadata>\n" + 
+			"                </featureType>".trim();
+	}
+
+	private String getStyleCreationPayload() {
+		return 
+			"                <StyledLayerDescriptor version=\"1.0.0\" xmlns=\"http://www.opengis.net/sld\">\n" + 
```

Comment:
Same comment as above. Do we want this as an XML file? I can see both arguments.

Reply:
Done.